pkg/storaged/pages.jsx

```diff
@@ -501,7 +501,7 @@ export const PageTable = ({ emptyCaption, aria_label, pages, crossrefs, sorted,
 
         if (typeof size === "number") {
             if (narrow)
-                size = fmt_size(size);
+                size = <span className="usage-text">{fmt_size(size)}</span>;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19685-20231204-165545-0123808a-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)